pii_recognition/evaluation/pipeline_test.py

```diff
@@ -0,0 +1,18 @@
+from unittest.mock import patch
+
+from pytest import raises
+
+from .pipeline import get_recogniser
+
+
+@patch(
+    "pii_recognition.evaluation.pipeline.recogniser_registry",
+    new={"fake_model": "fake_value"},
+)
+def test_get_recogniser():
+    actual = get_recogniser("fake_model")
+    assert actual == "fake_value"
+
+    with raises(KeyError) as err:
+        get_recogniser("non_exist_model")
+    assert str(err.value) == "'non_exist_model'"
```

Comment:
same here. more explicit error message pls. `recognizer requested is not registered`

Reply:
Current setting is to use registry as a dictionary, i.e.,
```
{"model_1": model_1_imp_class, "model_2": model_2_imp_class}
{"tokeniser_1": tokeniser_1_imp, "tokeniser_2": tokeniser_2_imp}
```
and all error messages are inherited from `dict` as well, the full message of a dictionary error you will see in terminal is `KeyError: model_3` or `KeyError: tokeniser_3`.

To implement more meaningful changes have to do something below
```
class Regsitry( meta_class=dict ):
    def __getitem__( self, key ):
        try:
            return super( ).__getitem__( key )
        except KeyError as e:
            raise MyErrorMessage( e.message )
```
I guess the question is that do users expect a `KeyError` message when using a dict-like object, or do users actually are expecting a meaningful custom error instead.